internal/expr/parser.go

```diff
@@ -302,14 +302,14 @@ func (p *Parser) skipName() bool {
 //  - bool == false, err == nil
 //		The construct was not the one we are looking for
 
-// parseIdentifierAsterisk parses a name made up of only nameBytes or of a
-// single asterisk. On success it returns the parsed string and true. Otherwise,
-// it returns the empty string and false.
-func (p *Parser) parseIdentifierAsterisk() (string, bool) {
+// parseIdentifierAsterisk parses a name made up of only nameBytes or of a single asterisk.
+// It returns an error (always nil) so it may be used with parseList.
```

Comment:
It is an implementation detail, but I think its good to make it clear that the returned error here can be safely ignored, since thats very much unexpected behaviour.